pkg/virt-launcher/virtwrap/manager.go

```diff
@@ -663,13 +663,42 @@ func (l *LibvirtDomainManager) getGuestTimeContext() context.Context {
 
 // FinalizeVirtualMachineMigration apply all post-migration changes on the domain.
 func (l *LibvirtDomainManager) FinalizeVirtualMachineMigration(vmi *v1.VirtualMachineInstance) error {
+	domainName := api.VMINamespaceKeyFunc(vmi)
+	virDomain, err := l.virConn.LookupDomainByName(domainName)
+	if err != nil {
+		return err
+	}
+	defer virDomain.Free()
+
+	if err := l.hotPlugHostDevices(vmi, virDomain); err != nil {
+		return err
```

Comment:
Since the hotplug is "nice to have" (the migration doesn't fail in case the some of the attachements failed), I believe we shouldn't return an error here and setting the `GuestTime` even in case the `hotPlugHostDevices` had an error.

Reply:
I changed it so it will log the error instead